### ISSUE TITLE
Padding and alignments of buttons, etc are weird when using styled + tailwind

### ISSUE THREAD

**Issue by elburro1887** [primefaces/primereact-examples#34]
When using Prime React with Styled + Tailwind options (Vite and Nextjs, for example), the padding on the button and the alignment of the icons/ other components is different than what is shown on the documentation page.

**When using the original values** in `vite-styled-tailwind/src/index.css`:
````
font-size: 16px;
line-height: 24px;
````

The button and the input field have excessive padding, radio buttons look normal.

<img width="391" alt="Screenshot 2024-07-30 at 00 21 33" src="https://github.com/user-attachments/assets/46ba39a9-6d09-4d95-bae5-9ca84246479e">

**Now I change the values** in in `vite-styled-tailwind/src/index.css`:
````
font-size: 14px;
line-height: 18px;
````
to match `font size: 14px` used the Button Component documentation page (https://primereact.org/button/). `line-height` is not used there, interestingly enough.

As one can see, the buttons look different, also the alignment of the radio buttons and the icon inside the button is off:

<img width="669" alt="Screenshot 2024-07-30 at 00 23 37" src="https://github.com/user-attachments/assets/1b615248-6d1e-465e-86fb-e8c87309eac6">

There seems to be some issue here where Tailwind is interfering with the layout, can anyone help out? Thanks.

**Comment by melloware** [comment]
Yeah this is tricky with Tailwind + Styled mode.  There are lots of conflicting CSS I am sure.

**Comment by elburro1887** [comment]
What would you suggest? Should I remove Tailwind altogether, or use Unstyled mode?

**Comment by melloware** [comment]
well you miht be able to debug it and see what is causing the padding issues and recommend a fix to PrimeReact.

**Comment by meowzhin** [comment]
Can you provide a stackblitz so I can take a look on the problem?

**Comment by elburro1887** [comment]
The code I tested is the one from this repo:
https://github.com/primefaces/primereact-examples/tree/main/vite-unstyled-tailwind

**Comment by melloware** [comment]
I just updated the sample to Tailwind 4.
